docs/source/WritingTests.rst

```diff
@@ -960,6 +960,10 @@ Avocado offers some options for the test writers to skip a test:
 Test ``skip()`` Method
 ----------------------
 
+.. warning:: `self.skip()` will be deprecated by the end of 2017.
```

Comment:
I don't mind the current, I'm curious whether I remember the rules correctly and what the native say about it as international English is sometimes different from the Oxford one...